todo_app/app.py

```diff
@@ -8,8 +10,16 @@
 
 @app.route('/')
 def index():
-    return 'Hello World!'
+    return view()
+
+@app.route('/add', methods = ['POST'])
+def add():
+    add_item(request.form.get('title'))
+    return view()
```

Comment:
This should ideally redirect back to the index route. Atm if you submit a todo and refresh the same todo will be added (probably not what a user would expect!)